core/src/test/scala/integration/kafka/admin/TopicCommandIntegrationTest.scala

```diff
@@ -789,9 +756,9 @@ class TopicCommandIntegrationTest extends KafkaServerTestHarness with Logging wi
       }
       val output = TestUtils.grabConsoleOutput(
         topicService.describeTopic(new TopicCommandOptions(Array("--under-min-isr-partitions"))))
-      val rows = output.split("\n")
-      assertTrue(rows(0).startsWith(s"\tTopic: $underMinIsrTopic"))
-      assertTrue(rows(1).startsWith(s"\tTopic: $offlineTopic"))
+      val rows = output.split(System.lineSeparator())
```

Comment:
Can we extract `System.lineSeparator()` into a class constant as well so that we don't have to make System calls every time to figure out this information? This isn't going to change throughout the tests in this test class.

Reply:
Done.